app/frontend/src/authConfig.ts

```diff
@@ -207,6 +207,7 @@ export const getToken = async (client: IPublicClientApplication): Promise<string
         .then(r => r.accessToken)
         .catch(error => {
             console.log(error);
+            window.alert(error);
```

Comment:
I don't know if there are situations where this will be annoying? Do we ever have errors that we're okay with ignoring?

Reply:
i would remove this personally